core/model/modx/processors/search/search.class.php

```diff
@@ -159,20 +159,22 @@ public function searchResources()
         }
 
         $c = $this->modx->newQuery('modResource');
+        $c->leftJoin('modTemplate','modTemplate','modResource.template=modTemplate.id');
+        $c->select($this->modx->getSelectColumns('modResource','modResource').", modTemplate.icon as icon");
```

Comment:
This is quite confusing to read - please just split it up into two `$c->select` lines.